hw3/ThronesAPI/src/CharacterPage.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from "react";
+import Searchbar from "./Searchbar";
+import { Container, Card } from "react-bootstrap";
+import axios from "axios";
+
+const url = "https://thronesapi.com/api/v2/Characters";
+
+const CharacterPage = () => {
+  const [characters, setCharacters] = useState([]);
+  const [searchResults, setSearchResults] = useState([]);
+
+  useEffect(() => {
+    const fetchCharacters = async () => {
+      try {
+        const response = await axios.get(url);
+        const apiData = response.data;
+        setCharacters(apiData);
+      } catch (error) {
+        console.log(error);
+      }
+    };
+    fetchCharacters();
+  }, []);
+
+  return (
+    <div>
+      <Searchbar characters={characters} setSearchResults={setSearchResults} />
+
+      {searchResults.map((character, index) => (
+        <Container style={{ display: "flex", justifyContent: "center" }}>
+          <Card
+            key={index}
+            className="py-3"
+            style={{ width: "24rem", justifyContent: "center" }}
+          >
+            <Card.Body>
+              <Card.Img
+                className="py-3"
+                variant="top"
+                src={character.imageUrl}
+                alt={character.fullName}
+              />
+              <Card.Title style={{ textAlign: "center" }}>
```

Comment:
I see you using inline styles in few places. There are bootstrap classes that you can replace it with like `text-center` for this property.